src/mediatype-icon.ts

```diff
@@ -13,7 +13,11 @@ import { webIcon } from './assets/img/icons/mediatype/web';
 
 @customElement('mediatype-icon')
 export class MediatypeIcon extends LitElement {
-  @property({ type: String }) mediatype = '';
+  @property({ type: String }) mediatype: string | undefined;
+
+  @property({ type: Boolean }) showText = false;
+
+  // @property({ type: Boolean }) isColored = false;
```

Comment:
You can remove this since it's not being used right now

Reply:
removed